tasks/main.yml

```diff
@@ -81,6 +81,13 @@
   sudo: yes
   template: src=kube-controller-manager.yaml dest={{kube_manifests_dir}}/kube-controller-manager.yaml
 
+- name: create scheduler config policy file
+  sudo: yes
+  template: src={{kube_scheduler_policy_config_file}} dest={{kube_scheduler_policy_config_dir}}/{{kube_scheduler_policy_config_file}}
+  when: 
+    - kube_scheduler_policy_config_dir is defined
+    - kube_scheduler_policy_config_file is defined
```

Comment:
You should add defaults for kube_scheduler_policy_config_dir & kube_scheduler_policy_config_file in defaults/main.yml and change these checks to: 
  - variable != ""